Tests/SwiftCurrent_SwiftUITests/AnyFlowRepresentableViewTests.swift

```diff
@@ -15,6 +15,13 @@ import SwiftCurrent
 
 @available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
 final class AnyFlowRepresentableViewTests: XCTestCase, View {
+    override func tearDownWithError() throws {
```

Comment:
This is my attempt to solve an issue we had. Short version: When you nested `inspect` calls too deep we weren't waiting for all of them. So I've set up this queue of expectations to wait on. This solution does work but comes with the drawback of having to put it in front of every test that uses the inspection extensions on `WorkflowItem`. 

I might've been able to hack something together that just did it for all tests, but that would make debugging harder.

Reply:
We could extract it into an extension on XCTestCase that is called in the `tearDownWithError()`.  This will decrease the duplication but still allow each TestCase to decide if it will do this.